app/lib/efile/ny/it201.rb

```diff
@@ -60,19 +60,19 @@ def calculate
         set_line(:IT201_LINE_40, :calculate_line_40)
         set_line(:IT201_LINE_43, :calculate_line_43)
         set_line(:IT201_LINE_44, :calculate_line_44)
-        set_line(:IT201_LINE_45, :calculate_line_45)
         set_line(:IT201_LINE_46, :calculate_line_46)
         set_line(:IT201_LINE_47, :calculate_line_47)
         set_line(:IT201_LINE_47A, :calculate_line_47a)
         set_line(:IT201_LINE_48, :calculate_line_48)
         set_line(:IT201_LINE_49, :calculate_line_49)
         set_line(:IT201_LINE_52, :calculate_line_52)
         set_line(:IT201_LINE_54, :calculate_line_54)
-        set_line(:IT201_LINE_54B, :calculate_line_54b)
+        # TODO: are we supporting MCTMT lines 54a-54e?
         set_line(:IT201_LINE_58, :calculate_line_58)
         set_line(:IT201_LINE_59, @intake, :sales_use_tax)
         @it227.calculate
-        set_line(:IT201_LINE_60E, -> { @lines[:IT227_PART_2_LINE_1].value })
+        # TODO: this is always 0, should we know the real value?
+        set_line(:IT201_LINE_60, -> { @lines[:IT227_PART_2_LINE_1].value })
```

Comment:
2023 form changed this from 60E to 60. i added the todo to verify that it's ok this is always 0